addons/mail/static/src/discuss/call/common/rtc_service.js

```diff
@@ -210,7 +210,8 @@ export class Rtc extends Record {
     timeouts = new Map();
     /** @type {Map<number, number>} timeoutId by sessionId for download pausing delay */
     downloadTimeouts = new Map();
-    iceServers = Record.attr(DEFAULT_ICE_SERVERS, {
+    /** @type {DEFAULT_ICE_SERVERS} */
```

Comment:
```suggestion
    /** @type {[[urls: string]: string[]}[]} */
```

For now at least, the script cannot locate other symbols and deduce their types so we have to type it explicitly :(